fileio/ft_read_event.m

```diff
@@ -514,6 +514,10 @@
               event(end  ).value    = tok{2};
               event(end  ).sample   = str2num(tok{3});
               event(end  ).duration = str2num(tok{4});
+              if size(tok, 2) >= 6
+                event(end).timestamp = datetime(tok{6}, 'InputFormat', ...
```

Comment:
please put this on a single line, without the `...`

Reply:
Done, just for my culture , why the `...` is depricated?